build/source/driver/multi_driver.f90

```diff
@@ -276,7 +283,21 @@ program multi_driver
 integer(i4b)                     :: fileHRU                    ! number of HRUs in the input file
 integer(i4b)                     :: iRunMode                   ! define the current running mode
 character(len=128)               :: fmtGruOutput               ! a format string used to write start and end GRU in output file names
- ! version information generated during compiling
+! timing information
```

Comment:
along the lines of Joe's comment, this could all be abstracted into a timing module.  I have a timer object implemented here if you are interested: 
https://github.com/gutmann/coarray_icar/blob/develop/src/objects/timer_interface.f90
https://github.com/gutmann/coarray_icar/blob/develop/src/objects/timer_implementation.f90
then you just have a timeGRU timer object, and call timeGRU%start() and timeGRU%stop(), timeGRU%get_time() (or as_string())

Reply:
Yes, this would be nice in principle. I will not address this as part of the current PR.